imports/plugins/included/email-smtp/server/startup.js

```diff
@@ -0,0 +1,4 @@
+import appEvents from "/imports/node-app/core/util/appEvents";
+import sendSMTPEmail from "./util/sendSMTPEmail";
+
+appEvents.on("sendEmail", sendSMTPEmail);
```

Comment:
@dancastellon Check out `/imports/plugins/core/cart/server/no-meteor/startup.js`. This is the preferred way to do a startup function and call `.on` in there. Then you do

```js
functionsByType: {
    startup: [startup]
  },
```

As seen in `/imports/plugins/core/cart/register.js`

The way you're doing it here does work, but by registering it as a plugin startup function, you can be sure that it doesn't happen until AFTER all of the core startup code is loaded.

Reply:
@aldeed All set here